proposals/OpaqueReflect.md

```diff
@@ -41,22 +41,37 @@ There is no specific syntax for opaque reflection. Instead, it takes place when
 
 For each unlifted free variable, it introduces a measure for it. For each measure introduced, we link the measure to the original function by strengthening the post-condition of the function, using the same pipeline as for reflection. We strengthen the post-condition by saying that the result `vv` is equal to the application of the measure to the arguments.
 
-If the user wishes to see what functions were opaque reflected, they can use the `--dump-opaque-reflections` pragma. It will list in the stdout the functions for which a measure was introduced automatically by opaque reflection.
+For instance,
 
-For instance, 
-
-`` `
+```Haskell
 {-@ assume GHC.List.filter :: p:(a -> GHC.Types.Bool) -> xs:[a] -> {v: [a] | len(v) <= len(xs) } @-}
 ```
 
 Would become:
 
-`` `
+```Haskell
 {-@ assume GHC.List.filter :: p:(a -> GHC.Types.Bool) -> xs:[a] -> {v: [a] | len(v) <= len(xs) && v = GHC.List.filter p xs} @-}
 ```
 
 The measure can bear the same name as the original function, since the symbol was unbounded in the logic before (otherwise we would not perform opaque reflection in the first place). Whence `filter` in the post-condition truly refers to the measure and not the Haskell function.
 
+If the user wishes to see what functions were opaque reflected, they can use the `--dump-opaque-reflections` pragma. It will list in the stdout the functions for which a measure was introduced automatically by opaque reflection. An example output for the following code is:
+
+```Haskell
+{-@ LIQUID "--reflection"      @-}
+{-@ LIQUID "--dump-opaque-reflections"      @-}
+
+module OpaqueRefl06 where
+
+{-@ reflect keepEvens @-}
+keepEvens :: [Int] -> [Int]
+keepEvens = filter even
+```
+
+```
+Opaque reflections: [GHC.Internal.List.filter,GHC.Internal.Real.even]
```

Comment:
Maybe printing the list one name per line is easier to read. If it is alphabetically sorted, it would be good to say so here as well.